### DISCUSSION TITLE
VarAC QSO app works

### DISCUSSION THREAD

**Discussion by SpudGunMan** [WheezyE/Winelink#45] [General]
https://www.varac-hamradio.com/forum/varac-discussion-forum/running-on-wine-raspberry64

VarAC Chat/QSO app works with this script I mentioned it in the linked article. FYI to this team.

Also I am Rpi4 bullseye64 and working wonderfully awesome work on this project.

**Comment by WheezyE** [comment]
Thanks for the heads-up. I'm sort of amazed that it works. I'd been getting weird behavior with the app (frozen screen) so hadn't included it yet. I would like to use a newer box86 at some point that might have less errors with this app, but I'm stuck at an older box86 that has TCP/IP support for VARA at the moment.

Sorry I didn't see this until now! I didn't think to check my Discussions tab until today

**Comment by WheezyE** [comment]
We've fixed TCP/IP support in VARA, so I can pursue this again, but I have never been able to get this working and am puzzled as to how others were able to

**Reply by SpudGunMan** [reply]
Issues with varac?

**Reply by WheezyE** [reply]
Yes VarAC debugging can continue now that we aren't stuck with using older box86 anymore.

**Reply by SpudGunMan** [reply]
Ahh yes nice! I also had some chats with author of varac he had some ideas to help wine functionality :)

**Comment by WheezyE** [comment]
I'm going to try to consolidate debugging, responses, and updates into this issue tracker for now: https://github.com/WheezyE/Winelink/issues/27
